api/core/rag/extractor/unstructured/unstructured_epub_extractor.py

```diff
@@ -17,7 +18,7 @@ class UnstructuredEpubExtractor(BaseExtractor):
     def __init__(
         self,
         file_path: str,
-        api_url: str = None,
+        api_url: Optional[str],
```

Comment:
The change to the __init__ method incorrectly makes the api_url parameter required by omitting the default value of None. This changes the behavior of the class constructor, which originally allowed api_url to be optional.

Reply:
Sorry for missing the default value None. Fixed.